IOS_SETUP.md

```diff
@@ -0,0 +1,47 @@
+# iOS Installation
+
+Background tasks on iOS work very differently than on Android.  
+Before anything, make sure you've added the following key to your project's `Info.plist`:
+```
+<key>UIBackgroundModes</key>
+  <array>
+    <string>fetch</string>
+  </array>
+</key>
+```
+
+Inside your app's delegate `didFinishLaunchingWithOptions` set your desired _minimumBackgroundFetchInterval_ in your app's delegate's:
+
+//TODO: provide a complete class
```

Comment:
Could we maybe have 1 complete example.  
Eg as an iOS noob I would not know where to place this exactly